packages/cardano-services/src/Program/ServiceNames.ts

```diff
@@ -4,5 +4,6 @@
  */
 export enum ServiceNames {
   StakePoolSearch = 'stake-pool-search',
-  TxSubmit = 'tx-submit'
+  TxSubmit = 'tx-submit',
+  Reward = 'rewards'
```

Comment:
```suggestion
  Rewards = 'rewards'
```

Reply:
Yes, sorry. I changed it in a new commit to then be squashed once approved, sounds good?

Reply:
Yes, that's the correct way to resolve.